dataeng/resources/answer-distribution.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Out of curiosity, what does this do?   I'm not familiar with "/usr/bin/env bash"...

Reply:
`/usr/bin/env whatever` respects the PATH set in the current environment.  It's basically less strict than an absolute `/bin/whatever` about which executable to use.  Ever since the `/usr/bin/bash` "scare" a few years back, `/usr/bin/env bash` won many followers who were afraid that `/bin/bash` would disappear, but realistically OSes just symlinked `/usr/bin/bash` to `/bin/bash`.

I'm indifferent about it---I just tried to make them all consistent.

Reply:
`#!/usr/bin/env python` is what really shows the true power of this style of shebang because it can respect your current virtualenv.

Reply:
Cool!   Thanks for the background -- good stuff to know.